en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -165,110 +148,120 @@ SilverStripe\Core\Injector\Injector:
       FilesystemAdapter: '%$SilverStripe\Assets\Flysystem\ProtectedAdapter'
 ```
 
-## Using constants and environment variables
+#### Using constants and environment variables
 
-Any of the core constants can be used as a service argument by quoting with back ticks "`". Please ensure you also quote the entire value (see below).
+The Injector configuration has the special ability to include core constants or environment variables. They can be used by quoting with back ticks "`". Please ensure you also quote the entire value (see below).
 
 ```yaml
-CachingService:
-  class: SilverStripe\Cache\CacheProvider
-  properties:
-    CacheDir: '`TEMP_DIR`'
+SilverStripe\Core\Injector\Injector:
+  CachingService:
+    class: SilverStripe\Cache\CacheProvider
+    properties:
+      CacheDir: '`TEMP_DIR`'
 ```
 
-Environment variables can be used in the same way:
+Environment variables are used in the same way:
 
 ```yml
-App\Services\MyService:
-    class: App\Services\MyService
-    constructor:
-      baseURL: '`SS_API_URI`'
-      credentials:
-        id: '`SS_API_CLIENT_ID`'
-        secret: '`SS_API_CLIENT_SECRET`'
+SilverStripe\Core\Injector\Injector:
+  App\Services\MyService:
+      class: App\Services\MyService
+      constructor:
+        baseURL: '`SS_API_URI`'
+        credentials:
+          id: '`SS_API_CLIENT_ID`'
+          secret: '`SS_API_CLIENT_SECRET`'
```

Comment:
Inconsistent indentation

Reply:
not 100% sure what you mean here sorry?

Reply:
seems like from the class down it's 2 spaces more than it should be